<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmsPickerWidget</class>
 <widget class="QWidget" name="AlarmsPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="AlarmsListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="reloadToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/icons8_synchronize_32px.png</normaloff>:/icons8_synchronize_32px.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/icons8_plus_math_32px_1.png</normaloff>:/icons8_plus_math_32px_1.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AlarmsListWidget</class>
   <extends>QListWidget</extends>
   <header>ui/util/alarmslistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
